### DISCUSSION TITLE
403 on HEAD requests, using s3 API

### DISCUSSION THREAD

**Discussion by Material-Scientist** [seaweedfs/seaweedfs#5930] [Q&A]
I'm able to make GET requests to the S3 API:
![image](https://github.com/user-attachments/assets/01924e8c-edcd-460a-a20b-0edff5423573)

But any HEAD requests fail with 403 (using same credentials):
![image](https://github.com/user-attachments/assets/273becf0-802c-491f-9fd3-a604a96a388c)

ACL:
```
      "actions": [
        "Read:default/*",
        "Tagging:default/*",
        "ReadAcp:default/*",
        "List:default/*",
        "Write:default/*",
        "Admin:default/*"
      ],
```

- version: 3.71_full

Am I doing something wrong or is this a bug?

**Comment by chrislusf** [comment]
could not tell. The GET and HEAD auth logic are the same.

**Reply by Material-Scientist** [reply]
Does it work on your setup?

**Reply by chrislusf** [reply]
you may need to have acl for "anonymous" user.

**Reply by Material-Scientist** [reply]
All the GET & HEAD requests above were made with admin credentials.

I also tried adding the following to identity.json:
```
{
      "name": "anonymous",
      "credentials": [
      ],
      "actions": [
      ],
      "account":  null
    }
```

But I'm still getting 403 on HEAD requests, while GET requests work with the admin credentials.
